src/encoded/server_defaults.py

```diff
@@ -35,6 +43,10 @@ def userid(instance, subschema):
 
 @server_default
 def now(instance, subschema):
+    return _now(instance, subschema)
+
+
+def _now(instance, subschema):
     # from jsonschema_serialize_fork date-time format requires a timezone
     return datetime.utcnow().isoformat() + '+00:00'
```

Comment:
```return pytz.UTC.localize(datetime.utcnow()).isoformat()```

(You'll have to add the pytz library to the imports.)

Reply:
Same issue about arguments to the internal function. And here I think you may want to call it `utc_now_str()` by analogy to the `utc_today_str()` I made elsewhere. We can collect them later. But this function doesn't need to be an internal function, it can be (eventually) an exported function of a utility library and for now, with a good name, a function that it's safe to use for other uses than just here, so it doesn't need the "_".